scripts/Dockerfile.integration_test

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+
+RUN apt-get install --yes git sudo wget curl make gcc apt-utils
+
+#install golang
+RUN wget "https://dl.google.com/go/$(curl https://golang.org/VERSION?m=text).linux-amd64.tar.gz" -O go.linux-amd64.tar.gz
```

Comment:
to make the builds reproducible, use Golang 1.15

Reply:
done